src/components/forms/NetworkDevicesForm/NetworkDeviceContent.tsx

```diff
@@ -0,0 +1,80 @@
+import ResourceLink from "components/ResourceLink";
+import type { FC } from "react";
+import type { LxdNicDevice } from "types/device";
+import NetworkDeviceAcls from "./NetworkDeviceAcls";
+import type { LxdNetwork } from "types/network";
+import NetworkSelector from "pages/projects/forms/NetworkSelector";
+import type { InstanceAndProfileFormikProps } from "../instanceAndProfileFormValues";
+import { supportsNicDeviceAcls } from "util/networks";
+
+interface Props {
+  readOnly: boolean;
+  project: string;
+  formik: InstanceAndProfileFormikProps;
+  device: LxdNicDevice;
+  index: number;
+  managedNetworks: LxdNetwork[];
+  network?: LxdNetwork;
+}
+
+const NetworkDeviceContent: FC<Props> = ({
+  readOnly,
+  project,
+  formik,
+  device,
+  index,
+  managedNetworks,
+  network,
+}: Props) => {
```

Comment:
```suggestion
}) => {
```
A bit shorter. I see some occasions of this, but it shouldn't be needed.